src/video/xml.rs

```diff
@@ -56,33 +57,45 @@ pub enum ParameterType {
 
 #[derive(Debug, Serialize)]
 pub struct ParameterBool {
+    #[serde(rename = "@name")]
     pub name: String,
-    #[serde(rename = "type")]
+    #[serde(rename = "@type")]
     pub cpp_type: String,
+    #[serde(rename = "@default")]
```

Comment:
why all this @ are necessary ?

Reply:
They change the serializer behavior to use the fields as parameters (as `<tag paramter="value"/> instead of sub-tags (as `<tag><subtag/></tag>`).

Reply:
Done